mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToHeader.java

```diff
@@ -53,7 +53,7 @@
  *
  * <pre>
  *     <code>
- *         &lt;mailet matcher=??? class=ICALToHeader&gt;
+ *         &lt;mailet match=??? class=ICALToHeader&gt;
```

Comment:
does that mean we break existing configuration?

Reply:
AFAIK the james code, [only the attribute `match` is considered](https://github.com/apache/james-project/blob/master/server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/lib/AbstractStateMailetProcessor.java#L312). It might have been called `matcher` before, but I don't know that.

Reply:
oh ok, I didn't understood. That's ok then.